doc/nrf/ug_nrf5340.rst

```diff
@@ -34,63 +34,42 @@ Network core
 ============
 
 The network core is an Arm Cortex-M33 processor with a reduced feature set, designed for ultra-low-power operation.
+This core should be used for radio communication and for real-time processing tasks involving low-level radio protocol layers.
 
-This core is used for radio communication.
-With regards to the nRF5340 samples, this means that the network core runs the radio stack and real-time procedures.
-
-Currently, the |NCS| provides the following solutions for the network core:
-
-* :ref:`ug_ble_controller` - both the SoftDevice Controller and the Zephyr Bluetooth LE Controller.
-* IEEE 802.15.4 radio driver - for Thread and Zigbee protocols.
-* :ref:`Multiprotocol <ug_multiprotocol_support>` - Thread and Zigbee protocols running in parallel with the :ref:`nrfxlib:softdevice_controller`.
-* Samples that directly use the radio peripheral.
-
-See `Network samples`_ for more information.
-
-In general, this core should be used for real-time processing tasks involving low-level radio protocol layers.
-
-The development kit name for the network core in Zephyr is ``nrf5340dk_nrf5340_cpunet``.
+The build target for the network core in Zephyr is ``nrf5340dk_nrf5340_cpunet``.
 
 Application core
 ================
 
 The application core is a full-featured Arm Cortex-M33 processor including DSP instructions and FPU.
-
-Currently, the |NCS| provides the following solutions for the application core:
-
-* High-level radio stack (the host part of the Bluetooth Low Energy, Thread, and Zigbee stacks) and application logic
-* Samples running only on the application core (for example, NFC samples with nRF5340 support)
-
-See `Application samples`_ for more information.
-
-In general, this core should be used for tasks that require high performance and for application-level logic.
+This core should be used for tasks that require high performance and for application-level logic.
 
 The M33 TrustZone divides the application MCU into secure and non-secure domains.
 When the MCU boots, it always starts executing from the secure area.
-The secure bootloader chain starts the :ref:`secure_partition_manager` sample, which configures a part of memory and peripherals to be non-secure and then jumps to the main application located in the non-secure area.
-
-In Zephyr, :ref:`zephyr:nrf5340dk_nrf5340` is divided into two different build targets:
-
-* ``nrf5340dk_nrf5340_cpuapp`` for the secure domain
-* ``nrf5340dk_nrf5340_cpuappns`` for the non-secure domain
-
-When built for the ``nrf5340dk_nrf5340_cpuappns`` build target, the :ref:`secure_partition_manager` sample is automatically included in the build.
+The |NCS| provides two alternatives for running applications from the non-secure area of the memory:
 
-.. tfm_support_start
+Secure Partition Manager (SPM)
```

Comment:
Could this be shared with the 91 docs somehow? It applies to both in the exact same way.

Reply:
Added to NCSDK-9187